onnx/defs/tensor/defs.cc

```diff
@@ -2202,15 +2202,20 @@ ONNX_OPERATOR_SET_SCHEMA(
             0,
             "output",
             "Tensor to copy input into.",
-            "T",
+            "V",
             OpSchema::Single,
             true,
             1,
             OpSchema::Differentiable)
         .TypeConstraint(
-            "T",
-            OpSchema::all_tensor_types_with_bfloat(),
-            "Constrain input and output types to all tensor types.")
+            "V",
+            [](){
+              auto t = OpSchema::all_tensor_types_with_bfloat();
+              auto s = OpSchema::all_tensor_sequence_types();
+              t.insert(t.end(), s.begin(), s.end());
+              return t;
+            }(),
+            "Constrain input and output types to all tensor and sequence types.")
```

Comment:
This is okay for this PR. But just wanted to point to the issue discussed here: https://github.com/onnx/onnx/issues/2324 ... in the long run, this is going to be limiting because this does not allow for sequences of sequences of tensors or maps, etc. It would be useful to extend this as discussed in the issue above. Any thoughts about this?